cli/tests/unit/stdio_test.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+import { unitTest, assertEquals } from "./test_util.ts";
+
+unitTest(async function stdioStdinRead() {
+  const nread = await Deno.stdin.read(new Uint8Array(0));
+  assertEquals(nread, 0);
+});
```

Comment:
These tests don't seem to be running...

Try changing the 0 to 42 and seeing if "cargo run std_tests" fails?

Reply:
[Seem to be running as far as I can tell](https://github.com/denoland/deno/pull/6126/checks?check_run_id=745481818#step:16:2289)?

But yes stdin tests do hang with a buffer bigger than 0 in byteLength.

Reply:
I would like to add some integration tests however, but seems the runner is not currently capable of sending something to stdin of the integration test?

Reply:
I'm worried it's not actually running the tests. Try intentionally breaking one of the asserts - does CI fail? That would give me confidence. I can look into this tomorrow...